webservices/docs.py

```diff
@@ -469,6 +476,14 @@
 - The communication is distributed within 60 days prior to a general election or 30 days prior to a primary election to federal office.
 '''
 
+# merge question????????????????
```

Comment:
@emileighoutlaw in merging the docs, I noticed we have 2 versions of electioneering, I like the first one because it is more specific, with days and says federal candidate instead of House and Senate, (because I see presidential electioneering so only house or senate doesn't make sense to me). Wondered if this came up for you before.

Data source:
http://www.fec.gov/finance/disclosure/electioneering.shtml